content/browser/renderer_host/render_process_host_impl.cc

```diff
@@ -2769,8 +2769,12 @@ void RenderProcessHostImpl::AppendRendererCommandLine(
   command_line->AppendSwitchASCII(
       service_manager::switches::kServiceRequestChannelToken,
       child_connection_->service_token());
+  // TODO: This guard should be remove after resolving workaround at
+  // ChildThreadImpl::Init.
+#if !defined(CASTANETS)
```

Comment:
This change means that the renderer's command line does not always include the kRendererClientId switch. However, if kRendererClientId does not exist, Castanets will not work with the normal path. What about the kEnableForking check instead of code disabling with !defined(CASTANETS)?

Reply:
You're right, we should keep the normal path. Please check the latest code.